docs/get-started/upgrading.md

```diff
@@ -24,7 +24,7 @@ The best way of doing this is wrapping any code that invokes a client class in a
 
 ```python
 async with Session() as session:
-    client = OrdersClient(session)
+    client = session.client('orders')
```

Comment:
I think instead of erasing `OrdersClient(session)` as a usage pattern across the docs, we should discuss the two usage patterns and when one should be used instead of the other, or mention that they both work equally. That way, when users come across `OrdersClient(session)` patterns in the source code and examples, they aren't totally confused because there is no documentation about it.

Reply:
@jreiberkyle agreed! I'll add a paragraph explaining that Session.client() is a user convenience and that we're using the other form internally and whenever we know what client we need before we create a session.